runelite-client/src/main/java/net/runelite/client/plugins/screenmarkers/ScreenMarkerPlugin.java

```diff
@@ -124,6 +124,8 @@ protected void startUp() throws Exception
 		clientToolbar.addNavigation(navigationButton);
 
 		mouseListener = new ScreenMarkerMouseListener(this);
+
+		mouseManager.registerMouseListener(mouseListener);
```

Comment:
You would need to unregister this mouse listener in shutdown.

Reply:
Done